internal/ci/ethereum_test

```diff
@@ -60,8 +60,9 @@ exit_handler() {
     # Clear all signal handlers to prevent handler loop
     trap - 1 2 3 15
     # Kill all child subprocesses
-    kill -- -$$ || true
+    pkill -P $$
```

Comment:
@j16r Switched to `pkill` so we only kill child subprocesses and exclude ourself for the `exit $errno` on 65.